test-packages/in-repo-addon-test.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const fs = require('fs-extra');
+const util = require('util');
+const rimraf = util.promisify(require('rimraf'));
+const chai = require('chai');
+const expect = chai.expect;
+const chaiFiles = require('chai-files');
+const dir = chaiFiles.dir;
+const file = chaiFiles.file;
+const path = require('path');
+const execa = require('execa');
+
+chai.use(chaiFiles);
+
+const BASE_PATH = path.join(__dirname, 'my-app-with-in-repo-addon');
+
+describe('in-repo addon coverage generation', function () {
+  this.timeout(10000000);
+
+  beforeEach(async function () {
+    await rimraf(`${BASE_PATH}/coverage*`);
+    await execa('git', ['clean', '-f', 'my-app-with-in-repo-addon'], { cwd: __dirname });
+    await execa('git', ['restore', 'my-app-with-in-repo-addon'], { cwd: __dirname });
```

Comment:
Seems vaguely odd to do `git` operations in tests, can you explain what this is cleaning up for?